web-client/iron-svelte-client/src/lib/popup-screen/popup-screen.svelte

```diff
@@ -12,15 +12,16 @@
     userInteractionService.subscribe((val) => {
         if (val != null) {
             userInteraction = val;
```

Comment:
note: It occurred to me that this code could be simplified

```suggestion
    userInteractionService.subscribe((userInteraction) => {
        if (userInteraction != null) {
```